README

```diff
@@ -15,7 +15,8 @@ library via FAudio#, which you can find in the 'csharp/' directory.
 
 Dependencies
 ------------
-FAudio depends solely on SDL2. FAudio never explicitly uses the C runtime.
+FAudio depends solely on SDL 2.0.9+ (Ubuntu 20.04+, Debian 10+, Fedora 30+)
```

Comment:
The SDL version is fine, I'd prefer not to have the distributions named since it's not comprehensive (which is impossible, admittedly).

Reply:
```suggestion
FAudio depends solely on SDL 2.0.9 or newer.
```

Reply:
Fair enough, patching now, just got bit today trying to compile this on an Ubuntu 18.04 box.